static/js/compose.js

```diff
@@ -414,22 +414,26 @@ exports.validate_stream_message_address_info = function (stream_name) {
     }
 
     var response;
-
+    var context={
+       'streamname': Handlebars.Utils.escapeExpression(stream_name),
+       'link' : "<a href='#streams/all'>",
+       'para' : '<p>',
+       'bold' : '<b>',
+       'closepara' : '</p>',
+       'closelink' : '</a>',
+       'closebold' : '</b>'
+     };
     switch (check_unsubscribed_stream_for_send(stream_name,
                                                page_params.narrow_stream !== undefined)) {
     case "does-not-exist":
-        response = "<p>The stream <b>" +
-            Handlebars.Utils.escapeExpression(stream_name) + "</b> does not exist.</p>" +
-            "<p>Manage your subscriptions <a href='#streams/all'>on your Streams page</a>.</p>";
+        response = i18n.t("__para__The stream __-bold____streamname____-closebold__ does not exist.__-closepara____-para__Manage your subscriptions __-link__on your Streams page__-closelink__.__-closepara__",context);
         compose_error(response, $('#stream'));
         return false;
     case "error":
         compose_error(i18n.t("Error checking subscription"), $("#stream"));
         return false;
     case "not-subscribed":
-        response = "<p>You're not subscribed to the stream <b>" +
-            Handlebars.Utils.escapeExpression(stream_name) + "</b>.</p>" +
-            "<p>Manage your subscriptions <a href='#streams/all'>on your Streams page</a>.</p>";
+        response=i18n.t("__para__You're not subscribed to the stream __bold____streamname____closebold__.__closepara____para__Manage your subscriptions __link__on your Streams page__closelink__.__closepara__",context);
```

Comment:
Same for this as well.

Reply:
![1](https://user-images.githubusercontent.com/25117249/33196115-69d3ea42-d103-11e7-81cd-03564451e69e.png)
still getting the error. it's not getting translated for some reason.

Reply:
Did you add the following in your context?
```
interpolation: {prefix: "{{", suffix: "}}"}  // We override the interpolation prefix and suffix here.
```

Reply:
Yeah.

Reply:
Can you try to debug in your console? This is what I get in my console.

![screen shot 2017-11-27 at 10 33 13 am](https://user-images.githubusercontent.com/641763/33252199-780ddac0-d35e-11e7-82cd-06800e95af9f.png)

Reply:
Sorry for the late reply. I had my exams. It's not working on my Firefox browser. Say's i18n not found